File: Python/Very Easy/simple_encoder.py
# https://edabit.com/challenge/5pBHzWb8Psot9EQNk

def simple_encoder(s):
    new_str=""
    s=s.lower()
    for letter in s:
        if s.count(letter)==1:
            new_str+='['
        elif s.count(letter)>=2:
            new_str+=']'
    return new_str

print(simple_encoder("Mubashir"), "[[[[[[[[")
print(simple_encoder("Matt"), "[[]]")
print(simple_encoder("Mu ba sh ir Ha ss an"), "[[][]]]]][[]]]]]]]][")

File: javascript/Easy/simpleEncoder.js

// https://edabit.com/challenge/hcXPiNgjXDhs8mGmq

function simpleEncoder(str) {
	const stringToSort=[...str.toLowerCase()];
    return stringToSort.map((el,i)=>{
        if(stringToSort.indexOf(el)==stringToSort.lastIndexOf(el)){
            return '['
        } else return ']'

    }).join('')
}
    



console.log(simpleEncoder("Mubashir"), "[[[[[[[[")
console.log(simpleEncoder("Matt"), "[[]]")
console.log(simpleEncoder("Mu ba sh ir Ha ss an"), "[[][]]]]][[]]]]]]]][")
console.log(simpleEncoder("HelEn"), "[][][")
console.log(simpleEncoder("Air  force"), "[[]]][[][[")
console.log(simpleEncoder("Pakistan"), "[][[[[][")
console.log(simpleEncoder("eD  aBiT"), "[[]][[[[")
console.log(simpleEncoder("Success"), "][]][]]")
console.log(simpleEncoder("iiiiii"), "]]]]]]")
// Mubashir